thermo_nerf/rgb_concat/concat_nerfacto_model.py

```diff
@@ -269,17 +272,19 @@ def get_image_metrics_and_images(
         metrics_dict = {"psnr": float(psnr.item()), "ssim": float(ssim)}  # type: ignore
         metrics_dict["lpips"] = float(lpips)
 
+        # Follow the same naming convention in nerfstudio, use "img" rather than
```

Comment:
Instead of making a special use case here, change the value of the RGB enum to img:

```
class Modality(Enum):
    RGB = "img"
```

is completely valid :)

Reply:
For concat-nerf, we overwrite the get_output() function of nerfacto model in nerfstudio. 
In concat-nerf model, we have to create and  use `output` and `image_dict`.

The naming convention for `output` in nerfstudio is output["rgb"].
The naming convention for `image_dict` in nerfstudio is image_dict["img"].

In this way, we cannot just set Modality.RGB.value as "img", as we are mainly using Modality.RGB.value for `output`.

Therefore, I guess the easiest way is to just use a special case here.

Reply:
I see now, that's annoying. Ok for now then :/